asllib/error.ml

```diff
@@ -149,6 +150,9 @@ module PPrint = struct
           pp_ty ty
           (pp_print_list ~pp_sep:pp_print_space pp_print_string)
           fields
+    | EmptySlice ->
+        fprintf f
+          "ASL Static Error:@ cannot@ slice@ with@ empty@ slicing@ operator"
```

Comment:
Do you mind adding that this may be due to incorrect invocation of a getter or a setter? Something like "expression is either a slice with an empty list of slices or an incorrect getter/setter invocation"